### DISCUSSION TITLE
[New] Eviter l'effet cul de sac

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#57] [Bonnes pratiques] [answered]
En complément des bonnes pratiques:
- Eliminer les fonctionnalités non essentielles
- Quantifier précisément le besoin
- Fluidifier le processus

Je trouverais intéressant d'avoir une bonne pratique qui explicite la chasse aux cul de sac. C'est notamment le cas dans des démarches administratives où les informations d'un site web peuvent ne pas être suffisant et provoquer un début d'utilisation du service et de ressources pour rien. La mise en place d'une alternative low tech ou low tech numérique permet de réduire ces effets.
A titre d'exemple, la souscription d'une assurance en ligne est parfois complexe et avoir la possibilité d'appeler un numéro gratuit ou de continuer la démarche en agence permet de limiter les échecs et donc une continuité dans le geste. A l'inverse, mettre en place un service de réservation de rendez-vous chez un médecin sans possibilité d'alternative téléphonique ou en cabinet (cf: ophtalmologues de la MEL) pose problème en cas d'urgence (trouver un rdv rapidement est empêché par le manque de contact humain).

**Comment by rlemaire** [comment]
« Effet cul de sac » n'est pas clair pour moi. Il faudrait sans doute parler de parcours client/utilisateur.

Par exemple « S'assurer que les parcours utilisateurs permettent de réaliser l'action prévue »



-

**Reply by ACTLEM** [reply]
Tu as raison. Ta formulation me plaît plus que la mienne :-)

**Reply by florinesueur** [reply]
J'aime beaucoup l'idée d'une alternative low tech. J'y vois aussi un autre angle de vue ou une autre bonne pratique qui consiste à prévenir l'utilisateur en amont des éléments indipensables et l'informer sur le temps nécessaire pour faire la procedure . Exemple : Pour ouvir un compte bancaire en ligne, il est indispensable d'avoir le scan ou photo d'une pièce d'identité, ...

**Reply by florinesueur** [reply]
Après relecture, est ce que ce n'est pas dans le champ d'action de la BP114 "Fluidifier le processus" de la discussion #114 ?

**Reply by ACTLEM** [reply]
Effectivement, cela se rapproche beaucoup, cela peut être pris comme un cas particulier. Après, dans "Fluidifier" c'est surtout réduire le temps de parcours que l'on veut atteindre, ici c'est surtout éviter que le parcours utilisateur ne s'arrête brusquement et que l'utilisateur soit bloqué (souvent car il ne répond pas à un cas nominal). Je pense surtout à tous les services sur le web qui arrivent en ce moment et qui ne peuvent se réaliser que sur le net sans aucune possibilité d'appel téléphonique, point d'accueil physique.
Etre obligé par exemple d'avoir un smartphone pour se connecter sur son compte en banque (l'application ne peut s'installer sur mon smartphone ou je n'en ai pas, comment je fais?), se connecter pour prendre rdv chez un ophtalmologue (j'ai une demande très particulière qu'il faut que je donne à mon ophtalmologue, comment je fais?) etc...

**Reply by florinesueur** [reply]
Oui, je vois la différence. Peut-être : Prévoir un parcours utilisateur alternatif low tech.
Je pense aussi aux formulaires sans fin et à la dernière étape ça bug sans information pour l'utilisateur, ni porte de sortie.

**Comment by Christophe-Cl** [comment]
Pour moi c'est plus une BP d'UX que d'écoconception. Avoir une approche low-tech ou écoconçue ne change rien.
Et c'est peut-être déjà couvert par la BP "Optimiser le parcours utilisateur" [#114](https://github.com/cnumr/best-practices/discussions/114)

**Comment by ACTLEM** [comment, marked as answer]
Note équipe projet: ok pour une BP spécifique sur l'absence de parcours alernatif

Issue créée: https://github.com/cnumr/best-practices/issues/185
